python/simulation.py

```diff
@@ -1255,6 +1299,7 @@ def stop_cond(sim):
             for func in step_funcs:
                 _eval_step_func(self, func, 'step')
             self.fields.step()
+            sys.stdout.flush()
```

Comment:
I thought #865 should have made this unnecessary. Are you seeing buffered output somewhere?

Reply:
Whoops, that slipped through. I'll fix that in the next commit. Thanks!